api-website/conf.py

```diff
@@ -28,10 +28,42 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
+<<<<<<< HEAD
```

Comment:
@chemik-bit  there is some merge conflict here?

Reply:
Yep, an old version was still present, fixed :)